src/tigerbeetle/cli.zig

```diff
@@ -1145,7 +1145,7 @@ fn parse_args_benchmark(benchmark: CLIArgs.Benchmark) Command.Benchmark {
         .account_count = benchmark.account_count,
         .account_count_hot = benchmark.account_count_hot,
         .account_distribution = benchmark.account_distribution,
-        .flag_history = benchmark.flag_history,
+        .flag_history = !benchmark.no_history,
```

Comment:
Let's invert the flag to `no_history` all the way down, and not only on the top CLI arguments struct.

Reply:
🙃 "fully" means the CLI structure, the Command structure and the Benchmark itself. Just to make it easier to grep. 

Which breaking change do you refer, other than the `tigerbeetle benchmark` CLI flag?